app/src/main/java/com/fastaccess/ui/modules/search/SearchUserActivity.kt

```diff
@@ -0,0 +1,118 @@
+package com.fastaccess.ui.modules.search
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.text.Editable
+import android.util.Log
+import android.view.View
+import android.widget.CheckBox
+import butterknife.*
+import com.fastaccess.R
+import com.fastaccess.helper.AnimHelper
+import com.fastaccess.helper.InputHelper
+import com.fastaccess.helper.Logger
+import com.fastaccess.ui.base.BaseActivity
+import com.fastaccess.ui.base.mvp.BaseMvp
+import com.fastaccess.ui.base.mvp.presenter.BasePresenter
+import com.fastaccess.ui.modules.search.repos.SearchReposFragment
+import com.fastaccess.ui.widgets.FontAutoCompleteEditText
+
+class SearchUserActivity : BaseActivity<BaseMvp.FAView, BasePresenter<BaseMvp.FAView>>() {
+
+    companion object {
+
+        @JvmStatic
+        val USERNAME = "username"
+        @JvmStatic
+        val SEARCH_TERM = "search"
+
+        @JvmStatic
+        fun getIntent(context: Context, username: String, searchTerm: String?): Intent {
+            val intent = Intent(context, SearchUserActivity::class.java)
+            intent.putExtra(USERNAME, username)
+            intent.putExtra(SEARCH_TERM, searchTerm)
+            return intent
+        }
+    }
+
+    @BindView(R.id.forkCheckBox) lateinit var forkCheckBox: CheckBox
+    @BindView(R.id.clear) lateinit var clear: View
+    @BindView(R.id.searchEditText) lateinit var searchEditText: FontAutoCompleteEditText
+
+    private var username = ""
+    private var searchTerm = ""
+    private var isFork = true
+    private lateinit var searchReposFragment: SearchReposFragment
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        ButterKnife.bind(this)
+
+        searchReposFragment = SearchReposFragment.newInstance()
+        supportFragmentManager.beginTransaction()
```

Comment:
please wrap this with 
`if(savedInstance == null)`

Reply:
@k0shk0sh Is there a reason to do this?
If I wrap the code
```kotlin
        searchReposFragment = SearchReposFragment.newInstance()
        supportFragmentManager.beginTransaction()
                .replace(R.id.containerFragment, searchReposFragment, "SearchReposFragment")
                .commit()
```
inside `if(savedInstanceState == null)`

I get an FC on orientation change saying that the fragment isn't initialized.
Without it, the activity works well.

Reply:
your code should be like this

```
        supportFragmentManager.beginTransaction()
                .replace(R.id.containerFragment, SearchReposFragment.newInstance(), "SearchReposFragment")
                .commit()
```

fun getSearchFragment() : SearchReposFragment? = return AppHelper.getFragmentByTag(supportFragmentManager, "SearchReposFragment")